code/clue.js

```diff
@@ -1,20 +1,143 @@
 // STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
 
-const mrGreen = {
+let mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
   color: 'green',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  favouriteWeapon: 'bat'
 }
 
+
+let professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: ' Crazy professor with large dreams ',
+  age: 40,
+  image: 'assets/plum.png',
+  occupation: 'Professor',
+  favouriteWeapon: 'rope'
+
+}
+
+let missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'People always underestimate her, they dont know her occupation',
+  age: 35,
+  image: 'assets/scarlet.png',
+  occupation: 'Undercover spy',
+  favouriteWeapon: 'candlestick'
+}
+
+let mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She is very rich',
+  age: 43,
+  image: 'assets/peacock.png',
+  occupation: 'Interior Designer',
+  favouriteWeapon: 'pistol'
+}
+
+let colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'loves the military',
+  age: 65,
+  image: 'assets/mustard.png',
+  occupation: 'Retired',
+  favouriteWeapon: 'axe'
+}
+
+let mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'She became a widower and got her freedom back',
+  age: 63,
+  image: 'assets/white.png',
+  occupation: 'Globetrotter',
+  favouriteWeapon: 'poision'
+}
+
+//console log test 
+console.log(mrsWhite.description)
+
+
+
+
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
+  id: 'rope',
   name: 'rope',
-  weight: 10
+  weight: 6,
+  color: 'brown'
+}
+
+const knife = {
+  id: 'knife',
+  name: 'knife',
+  weight: 5,
+  color: 'dark grey'
+
+}
+
+const candleStick = {
+  id: 'candlestick',
+  name: 'candlestick',
+  weight: 6,
+  color: 'rosegold'
+}
+
+const dumbBell = {
+  id: 'dumbbell',
+  name: 'dumbbell',
+  weight: 10,
+  color: 'grey'
+}
+
+const poision = {
+  id: 'poision',
+  name: 'poision',
+  weight: 3,
+  color: 'green'
+}
+
+const axe = {
+  id: 'axe',
+  name: 'axe',
+  weight: 10,
+  color: 'silver'
+}
+
+const bat = {
+  id: 'bat',
+  name: 'bat',
+  weight: 7,
+  color: 'black'
+}
+
+const trophy = {
+  id: 'trophy',
+  name: 'trophy',
+  weight: 9,
+  color: 'gold'
+}
+
+const pistol = {
+  id: 'pistol',
+  name: 'pistol',
+  weight: 8,
+  color: 'black'
```

Comment:
I can see that you are starting to use ; later on in the code. This is something that is really hard for me to do and its something that my developer colleagues recommend me to improve on. Remember what the teachers said "Everything or nothing!!"